main/res/raw/changelog_base.md

```diff
@@ -11,6 +11,7 @@
 - New: Attributes overview (see Manage Caches => Attributes overview)
 - New: Add import from bookmark lists (GC premium only)
 - New: Invert sort-order on long click on sort bar
+- Change: Also perform automatic sorting by distance for lists containing cache series with more than 50 caches (up to 500)
```

Comment:
FYI: Including the changelog message in the PR will lead to merge conflicts, if someone else creates a PR adding a change notice in the same line. Had that in the past frequently, and reverted procedure to adding the changelog messages later on.

Reply:
mmh...technically correct. But one could still remove it when it really happens to produce conflicts. IMHO its less likely than one might thing (same line, both PR opened).
Adding later one makes it more likely, that things are forgotten.

Reply:
PR do not necessarily have to be open at the same time, it will also happen, when you haven't updated & rebased your local copy while working on your PR. It's sort of a "single point of failure", as changes in a PR are typically in different places, but the changelog will always be the same file. (We have worked like this a while ago, and it lead to quite a few merge problems back then.)

But anyway, for a single PR as this it won't hurt, and we can monitor how it will work if we follow this way more regularly.